specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -1368,6 +1368,379 @@
+            "schema": {
+              "$ref": "#/definitions/SqlContainerListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      },
+      "put": {
+        "operationId": "DatabaseAccounts_CreateSqlContainer",
+        "x-ms-examples": {
+          "CosmosDBSqlContainerCreate": { "$ref": "./examples/CosmosDBSqlContainerCreate.json" }
+        },
+        "description": "Creates an Azure Cosmos DB SQL container",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseRidParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "createSqlContainerParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SqlContainerCreateUpdateParameters"
+            },
+            "description": "The parameters to provide for the current SQL container."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "The SQL container create operation was successful",
+            "schema": {
+              "$ref": "#/definitions/SqlContainer"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/apis/sql/databases/{databaseRid}/containers/{containerRid}": {
+      "get": {
+        "operationId": "DatabaseAccounts_GetSqlContainer",
+        "x-ms-examples": {
+          "CosmosDBSqlContainerGet": { "$ref": "./examples/CosmosDBSqlContainerGet.json" }
+        },
+        "description": "Gets the SQL container under an existing Azure Cosmos DB database account.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/databaseRidParameter"
+          },
+          {
+            "$ref": "#/parameters/containerRidParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The SQL container property was retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/SqlContainer"
+            }
+          }
+        }
+      },
+      "put": {
```

Comment:
Missing a PATCH api to support updates (required for tags, recommended for other mutable properties)Missing a PATCH api to support updates (required for tags, recommended for other mutable properties)

Reply:
We do not support PATCH

Reply:
This is required for tags. You can specify a request body that just exposes "tags" as a patchable property